### ISSUE TITLE
md-radio-group doesn't work with Talkback if it has an aria-labelledby attribute

### ISSUE THREAD

**Issue by calebegg** [angular/material#2314]
Here's a Codepen demonstrating the issue:

http://codepen.io/calebegg/pen/azeWga

The control can't be used with Talkback on Android (it won't drill down into the individual options).

The simple demo below corresponding to all of the relevant parts of the angular-material demo does work with Talkback, so I think this is a bug with angular-material.

**Comment by marcysutton** [comment]
@calebegg I don't think this is an issue. Angular Material dynamically copies text content inside of `<md-radio-button>` to `aria-label`, which labels each individual radio button. They read aloud just fine and are usable in Talkback in Chrome on Android 4.4.4. Which browser and OS version are you using?

Additionally, there is a big difference between the Angular Material example and the "simple demo" in your Codepen, the latter of which uses `role=radio` but doesn't add any kind of labeling to each button. Native radio inputs take individual labels, and a group could be described further with fieldset/legend or `aria-describedby`. If there are issues with either of those, we'll want to fix them for sure. But for now I'm going to close as `aria-labelledby` isn't the correct way to describe a radio group.

**Comment by calebegg** [comment]
aria-describedby has the same problem for me (I updated the codepen demo).

You can't reproduce it in the demo I provided? You were able to select and use the radio buttons using talkback (besides the middle one)?

I'm on Android 5.1 and Chrome 41.0.2272.96

**Comment by marcysutton** [comment]
@calebegg I just remembered I need to check up on this one again, so I've reopened it. Thanks again for reporting.
